### PR TITLE
Reinstate ID used in Smoke tests for Homepage

### PR DESCRIPTION
**Pull request by chao-xian**
[Related Smokey test affected](https://github.com/alphagov/smokey/blob/main/features/step_definitions/whitehall_steps.rb#L3)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by chao-xian** [comment]
Deployed this branch and Smokey passes on integration

**Review by owenatgov — COMMENTED** [review]
I'm regrettably pushing back on this. The old homepage was full of these id's that were only used for tests and the result was markup that probably didn't make a difference to the user. This is good in one sense but bad in another in that we're serving unclean markup and unclean semantics to the user. I'd rather keep it neat.

File: app/views/homepage/_government_activity.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<section class="homepage-section homepage-section__government-activity">
+<section class="homepage-section homepage-section__government-activity" id="departments-and-policy">
```
[the remark this answers is not in the document]
Comment (chao-xian) [reply app/views/homepage/_government_activity.html.erb:1]: Good suggestion! I'll amend

[the remark this answers is not in the document]
Comment (chao-xian) [reply app/views/homepage/_government_activity.html.erb:1]: Replaced with https://github.com/alphagov/smokey/pull/876 which is now merged

**Pull request closed without merging**